troposphere/static/js/components/projects/resources/instance/details/actions/InstanceActionsAndLinks.jsx

```diff
@@ -104,7 +104,8 @@ export default React.createClass({
 
         if (actions) {
             let keys = actions.pluck("key");
-            actionElements = this.actionSpecs().filter((a) => keys.includes(a.key));
+            actionElements = this.actionSpecs()
+            // actionElements = this.actionSpecs().filter((a) => keys.includes(a.key));
```

Comment:
Temporary change so any action can be performed on an instance during testing, not part of pr